<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehiclePage</class>
 <widget class="QWizardPage" name="VehiclePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>550</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Vehicle type selection&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; font-weight:600;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;To continue the wizard needs to know what type of vehicle that the OpenPilot controller board is going to be used with. This step is cruicial since most of the following configuration is unique per vehicle type.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;So go ahead and select the type of vehicle you want to create a configuration for.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>561</width>
     <height>181</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Supported vehicle types</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QToolButton" name="multirotorButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>100</width>
      <height>120</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Tricopter, Quadcopter, Hexacopter, Octocopter</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton { border: none }</string>
    </property>
    <property name="text">
     <string>Multirotor</string>
    </property>
    <property name="icon">
     <iconset resource="../wizardResources.qrc">
      <normaloff>:/setupwizard/resources/bttn-multi-up.png</normaloff>
      <normalon>:/setupwizard/resources/bttn-multi-down.png</normalon>
      <activeoff>:/setupwizard/resources/bttn-multi-down.png</activeoff>
      <activeon>:/setupwizard/resources/bttn-multi-down.png</activeon>:/setupwizard/resources/bttn-multi-up.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="fixedwingButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>100</width>
      <height>120</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Airplane, Sloper, Jet</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton { border: none }</string>
    </property>
    <property name="text">
     <string>Fixed wing</string>
    </property>
    <property name="icon">
     <iconset resource="../wizardResources.qrc">
      <normaloff>:/setupwizard/resources/bttn-plane-up.png</normaloff>
      <normalon>:/setupwizard/resources/bttn-plane-down.png</normalon>
      <activeoff>:/setupwizard/resources/bttn-plane-down.png</activeoff>
      <activeon>:/setupwizard/resources/bttn-plane-down.png</activeon>:/setupwizard/resources/bttn-plane-up.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="heliButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>100</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton { border: none }</string>
    </property>
    <property name="text">
     <string>Helicopter</string>
    </property>
    <property name="icon">
     <iconset resource="../wizardResources.qrc">
      <normaloff>:/setupwizard/resources/bttn-heli-up.png</normaloff>
      <normalon>:/setupwizard/resources/bttn-heli-down.png</normalon>
      <activeoff>:/setupwizard/resources/bttn-heli-down.png</activeoff>
      <activeon>:/setupwizard/resources/bttn-heli-down.png</activeon>:/setupwizard/resources/bttn-heli-up.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="surfaceButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>100</width>
      <height>120</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Car, Boat, U-Boat</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton { border: none }</string>
    </property>
    <property name="text">
     <string>Surface</string>
    </property>
    <property name="icon">
     <iconset resource="../wizardResources.qrc">
      <normaloff>:/setupwizard/resources/bttn-land-up.png</normaloff>
      <normalon>:/setupwizard/resources/bttn-land-down.png</normalon>
      <activeoff>:/setupwizard/resources/bttn-land-down.png</activeoff>
      <activeon>:/setupwizard/resources/bttn-land-down.png</activeon>:/setupwizard/resources/bttn-land-up.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../wizardResources.qrc"/>
 </resources>
 <connections/>
</ui>
